eos_validator.py

```diff
@@ -0,0 +1,51 @@
+
+# Check the TOKEN amount
+def check_balance(node_host="http://localhost:8888", snapshot_csv="./snapshot.csv", eos_total=1000000000):
+    with open(snapshot_csv, 'r') as fp:
+        account_onchain_total = 0
+        for onepersion in fp.readline()
+            account_name, account_amount, pub_key = json.loads(onepersion)['name'], json.loads(onepersion)['amount'],json.loads(onepersion)['pub'] 
+            if account_amount<1.0:
+                continue
+            account_balance = exec('cleos -u ' + node_host + ' get currency balance eosio.token ' + account_name)
+            account_net_delegated = exec('cleos -u ' + node_host + ' get account ' + account_name)['delegated_bandwidth']['net_weight']
+            account_cpu_delegated = exec('cleos -u ' + node_host + ' get account ' + account_name)['delegated_bandwidth']['cpu_weight']
```

Comment:
Here calls twice cleos get account on the same account, better save the output in a variable and then get the net_weight/cpu_weight from the variable, cut cleos calls by half

Reply:
yes, the code is just for demo, I will do what you suggested when realize.